io.openems.edge.meter.janitza.umg96rme/generated/buildfiles

```diff
@@ -0,0 +1 @@
+C:/Users/hueseyin.sahutoglu/git/repositories/powervoltagecharacteristic/io.openems.edge.meter.janitza.umg96rme/generated/io.openems.edge.meter.janitza.umg96rme.jar
```

Comment:
Do not add this file

Reply:
as we discussed, for deleted project generate itself, missed out when it staged